pkg/bindings/connection.go

```diff
@@ -91,9 +91,12 @@ func NewConnectionWithIdentity(ctx context.Context, uri string, identity string,
 	var connection Connection
 	switch _url.Scheme {
 	case "ssh":
-		port, err := strconv.Atoi(_url.Port())
-		if err != nil {
-			return nil, err
+		port := 22
+		if _url.Port() != "" {
+			port, err = strconv.Atoi(_url.Port())
```

Comment:
you could just do `port, _ =`

Reply:
sure, but I will leave the if { since atoi returns 0 on failure